### DISCUSSION TITLE
access navidrome server from android client. need help

### DISCUSSION THREAD

**Discussion by RayZ0rr** [containers/podman#14115] [Q&A]
I'm using podman in rootless mode instead of docker for the [docker version](https://www.navidrome.org/docs/installation/docker/) of [navidrome](https://www.navidrome.org/).

After running this command: 
```
podman run -d \
   --name navidrome \
   --restart=unless-stopped \
   -v /path/to/music:/music \
   -v /path/to/data:/data \
   -p 4533:4533 \ 
   -e ND_LOGLEVEL=info \
   docker.io/deluan/navidrome:latest
```
derived from the docker one given in their page:
```
docker run -d \
   --name navidrome \
   --restart=unless-stopped \
   --user $(id -u):$(id -g) \
   -v /path/to/music:/music \
   -v /path/to/data:/data \
   -p 4533:4533 \ 
   -e ND_LOGLEVEL=info \
   deluan/navidrome:latest
```

(`   --user $(id -u):$(id -g) \` is removed in podman version because it's rootless mode)

I can access navidrome web GUI at http://localhost:4533 or 0.0.0.0:4533 from the host where the command is run.

However, I cannot access the address http://<navidrom_host_ip_address>:4533 from my android client when I use the podman version. The docker version works fine. What change should I make? Please help.

**Comment by Luap99** [comment]
Since you run rootless podman cannot open the ports in the firewall. You need to open the ports manually, this depends on what firewall is used on your system/distro. If you use firewalld see the docs here: https://firewalld.org/documentation/howto/open-a-port-or-service.html
